### ISSUE TITLE
Bump docutils when sphinx unpins

### ISSUE THREAD

**Issue by rikardn** [pharmpy/pharmpy#347]
When sphinx has released its pin to docutils we should bump the version for docutils

Current version of sphinx: 4.2.0
Non-working version of docutils: 0.18

Upstream issue: https://github.com/sphinx-doc/sphinx/issues/9777

**Comment by rikardn** [comment]
Sphinx doesn't block this any longer, but there  is a warning:

```
.tox/py310/lib/python3.10/site-packages/docutils/transforms/__init__.py:173: in apply_transforms
    transform.apply(**kwargs)
.tox/py310/lib/python3.10/site-packages/jupyter_sphinx/ast.py:628: in apply
    for cell_node in self.document.traverse(JupyterCellNode):
.tox/py310/lib/python3.10/site-packages/docutils/nodes.py:225: in traverse
    warnings.warn('nodes.Node.traverse() is obsoleted by Node.findall().',
E   PendingDeprecationWarning: nodes.Node.traverse() is obsoleted by Node.findall().

FAILED tests/tools/test_frem.py::test_create_report - PendingDeprecationWarning: nodes.Node.traverse() is obsoleted by Node.findall().

```

**Comment by rikardn** [comment]
Fixed by e26d8977df916028b2f0c1dfbb17dec4df12266b
